tests/test_runner.py

```diff
@@ -1 +1,204 @@
-# -*- coding: utf8 -*-
+# -*- coding: utf-8 -*-
+import os
+import sys
+import unittest
+
+import invoke
+import mock
+from clint.textui import colored
+from frigg import projects
+from invoke.exceptions import Failure
+from invoke.runner import Result
+
+from frigg_runner.runner import Runner
+
+
+class RunnerTestCase(unittest.TestCase):
+
+    @mock.patch('frigg.projects.build_settings')
+    def test_runner_init(self, mock_run):
+        """
+        Test init of the runner class
+        """
+        runner = Runner(True, True)
+
+        self.assertTrue(runner.fail_fast)
+        self.assertTrue(runner.verbose)
+        self.assertEqual(runner.directory, os.getcwd())
+
+        projects.build_settings.assert_called_once_with(runner.directory)
+
+    def test_no_tasks(self):
+        """
+        No tasks, system exit.
+        """
+        def raise_runtime_error(*args, **kwargs):
+            raise RuntimeError
+
+        projects.build_settings = mock.Mock(side_effect=raise_runtime_error)
```

Comment:
Please use mock.patch-decorator instead. The side_effect argument can take an exception class or instance and raise it so `side_effect=RuntimeError` should give the same result.